R/caret.R

```diff
@@ -4,10 +4,10 @@
 #'  \pkg{rsample} and \pkg{caret}.
 #'
 #' @param object An `rset` object. Currently,
-#'  `nested_cv` is not supported.
-#' @return `rsample2caret` returns a list that mimics the
+#'  `nested_cv()` is not supported.
```

Comment:
Instead of backticks, can you use brackets so cross-linking happens? For example: 

```suggestion
#'  [nested_cv()] is not supported.
```